File: 351003/Mashevskiy/Lab1/src/main/java/by/tracker/rest_api/model/Tweet.java
package by.tracker.rest_api.model;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Tweet {
    private Long id;
    private Long creatorId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
}

File: 351002/Abrashin/publisher/src/main/kotlin/com/example/distcomp/model/Tweet.kt
package com.example.distcomp.model

import java.time.LocalDateTime

data class Tweet(
    var creatorId: Long? = null,
    var title: String? = null,
    var content: String? = null,
    var created: LocalDateTime? = null,
    var modified: LocalDateTime? = null,
    var stickers: List<Sticker> = emptyList()
) : BaseEntity()
